datafusion/wasmtest/Cargo.toml

```diff
@@ -37,11 +37,14 @@ crate-type = ["cdylib", "rlib",]
 # all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
 # code size when deploying.
 console_error_panic_hook = { version = "0.1.1", optional = true }
+datafusion = { path = "../core", default-features = false }
 
 datafusion-common = { workspace = true }
+datafusion-execution = { workspace = true }
```

Comment:
I double checked and this addition means that the wasm ci job now checks these crates
https://github.com/apache/arrow-datafusion/blob/8353a2ca2fd1f0ed5fc764b7463dfbcaa033ceef/.github/workflows/rust.yml#L192-L207

Here is the example CI run from this PR showing the new crates building in wasm https://github.com/apache/arrow-datafusion/actions/runs/7504130188/job/20430447784?pr=8843#step:6:194